lib/matplotlib/scale.py

```diff
@@ -667,14 +667,19 @@ class LogitScale(ScaleBase):
     """
     name = 'logit'
 
-    def __init__(self, axis, nonpos='mask'):
-        """
+    def __init__(self, axis, nonpos='mask', *, use_overline=False):
+        r"""
         *nonpos*: {'mask', 'clip'}
           values beyond ]0, 1[ can be masked as invalid, or clipped to a number
           very close to 0 or 1
+
+        use_overline: bool (default: False)
+            indicate the usage of survival notation (\overline{x}) in place of
+            standard notation (1-x) for probability close to one.
```

Comment:
How standard is this notation? Do you have an example of its use (e.g. on a publication)?  Just curious...

Reply:
This notation is very common in extreme value theory for survival function (1-cdf()). Generally we note $F$ the cdf, and $\overline{F}$ the survival. [For example, in the introduction](https://hal.archives-ouvertes.fr/hal-00942134/document). The use of the notation for numerical value exists, but it is not so common (personally I dislike this notation for numerical values).

 - For sets: \(A \cup \overline{A} = U\), with \(U\) the universe, the most common.
 - For survival function: \(F(x) + \overline{F}(x) = 1\), common, I use it.
 - For probability: \( p + \overline{p} = 1\), rare, I do not use it.

Reply:
If it's rare and you don't like it, do we really want to even implement it? (remember, any feature we implement means more code to maintain, more bug reports, etc.) I don't feel very strongly about it either way, though.

Reply:
Some colleagues of mine prefer use this notation in internal documents, but they change notation for publication. I ask them, having this notation is not a major issue. The concerned code is not large, it is deduplicated by the internal `_one_minus`, therefore I suggest keep this functionality. But I agree to remove it if asked.

Reply:
that's fine, you can leave it.